website/content/sdk/usage/basics/index.md

```diff
@@ -4,196 +4,390 @@ sidebar_position: 1
 description: The OpenBB SDK is a modern investment research platform for everyone. It is a Software Development Kit that allows you to access all the data and tools you need to make better investment decisions.
 keywords: [basics, installation, commands, menus, your own data, introduction, openbb sdk, explanation, basic usage, usage, how to, openbb terminal, importing, navigation, docstrings, function, charts, display]
 ---
-
 import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
 
-<HeadTitle title="Basics - SDK | OpenBB Docs" />
+`<HeadTitle title="Basics - SDK | OpenBB Docs" />`
 
-### Importing the SDK
+## Overview
 
-Now that you have the SDK installed, the first step is to import the OpenBB SDK in your preferred code editor. Nearly everything required to interact with any function from the OpenBB Terminal gets imported in one line. Begin a Python script or Notebook file with:
+The OpenBB SDK provides programmatic access to nearly all Terminal commands, allowing custom scripts and functions to power creations built on top of the core platform.  It is easy to use and as flexible as one's imagination.  The sections below will outline how to get started using it.  If it isn't already installed, follow the instructions to install via [PyPi (pip)](https://docs.openbb.co/terminal/installation/pypi) or from the [source code](https://docs.openbb.co/terminal/installation/source).
```

Comment:
So we should probably differentiate that the sdk itself (without all) can be installed pretty much anywhere?  I can just fire up a venv, run pip install openbb and I am good to go.

Reply:
mmmm...that would mean we are telling people to install the Terminal this way, and we still want people to install by following the instructions so they don't make up their own methods and complain when "pip install doesn't work".  

I think I have some install steps for Google CoLab that are now stable.

Reply:
Not sure if this is the ideal spot for these instructions... thoughs?